fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AlterTableClauseVisitor.java

```diff
@@ -106,8 +106,16 @@ public void analyze(AlterClause statement, ConnectContext session) {
     public Void visitCreateIndexClause(CreateIndexClause clause, ConnectContext context) {
         IndexDef indexDef = clause.getIndexDef();
         indexDef.analyze();
-        clause.setIndex(IndexFactory.createIndexFromDef(indexDef));
-
+        Index index;
+        // Only assign meaningful indexId for OlapTable
+        if (table.isOlapTableOrMaterializedView()) {
```

Comment:
In function `tryToAssignIndexId `, it ignore the effect by the BITMAP INDEX. I think that the compatibility strategy for Bitmap indexes should always set index_id to -1? It seems that the behavior is inconsistent between here and `tryToAssignIndexId `.

Reply:
Fix it.